src/App.tsx

```diff
@@ -44,32 +43,35 @@ function App() {
     sourceID,
   });
 
-  if (!isOnClientSide()) return null;
+  if (!isOnClientSide()) return <AppHead {...headProps} />;
```

Comment:
This was previously rendered by route components but since those have to be the same to avoid app remounting I had to move rendering `AppHead` to this component